src/server/js/routes/listing.js

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ *  Online store PWA sample.
+ *  Copyright 2017 Google Inc. All rights reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ *
+ */
+
+import {Router} from 'express';
+const router = new Router();
```

Comment:
I think we should not be creating a new router for every single route. Maybe we can put all of them in index.js?

Reply:
Convention seems to be:
```javascript
import {Router} from 'express';
const router = Router();
```

e.g.
https://www.terlici.com/2014/09/29/express-router.html
https://github.com/searsaw/express-routing-example/blob/master/routes/cars/index.js

Reply:
Ignore that comment, I don't know what I'm talking about.

My actual comment is that I feel each of these routing configurations will get larger as we do the actual implementation. I think it's prudent to separate them now even though they're trivial as we're just going to add more anyway.

Reply:
Keeping things separate for now, then.

Reply:
Definitively we should keep handlers in separate files. But should we have a new router for each of them?

Reply:
Ok. I still feel it's superfluous, but let's see how it goes.